tcms/rpc/api/forms/testrun.py

```diff
@@ -36,22 +36,13 @@ def populate(self, product_id):
         self.fields['build'].queryset = Build.objects.filter(product_id=product_id, is_active=True)
 
 
-class NewExecutionForm(BaseCaseRunForm):
-    assignee = forms.ModelChoiceField(queryset=User.objects.all(), required=False)
-    run = forms.ModelChoiceField(queryset=TestRun.objects.all())
-    case = forms.ModelChoiceField(queryset=TestCase.objects.all())
-
-    def clean_assignee(self):
-        data = self.cleaned_data.get('assignee')
-        if not data:
-            if self.cleaned_data.get('case') \
-                    and self.cleaned_data['case'].default_tester_id:
-                data = self.cleaned_data['case'].default_tester
-            elif self.cleaned_data.get('run') \
-                    and self.cleaned_data['run'].default_tester_id:
-                data = self.cleaned_data['run'].default_tester
+class NewTestExecutionForm(forms.ModelForm):
+    class Meta:
+        model = TestExecution
+        fields = '__all__'
 
-        return data
+    assignee = UserField(required=False)
+    tested_by = UserField(required=False)
```

Comment:
These 2 fields now can accept multiple values which needs tests (positive tests are fine). In the test case changed below I don't see scenarios for username/email for both of these fields.

Since they aren't required there are no test scenarios for them anyway. You can combine the scenarios together if you want.